src/helpers/helpers.canvas.js

```diff
@@ -44,112 +44,121 @@ var exports = module.exports = {
 		}
 	},
 
-	drawPoint: function(ctx, style, radius, x, y) {
-		var type, edgeLength, xOffset, yOffset, height, size;
+	drawPoint: function(ctx, style, width, height, x, y, isLineWidthZero) {
```

Comment:
what is `isLineWidthZero` used for here?

Reply:
For polygon (circle, rect, ....) if line width is 0, then no external line are displayed (no strike).
This is replication of the current behavior for the rect symbol. Not doing this may impact the current users upgrading to that new version even if they don't change anything to their code (regression).